libs/components/code-examples/src/lib/modules/filter-bar/filter-bar/basic/filter-modal-harness.ts

```diff
@@ -1,31 +1,82 @@
 import { HarnessPredicate } from '@angular/cdk/testing';
 import { SkyComponentHarness } from '@skyux/core/testing';
+import { SkyRadioGroupHarness, SkyRadioHarness } from '@skyux/forms/testing';
 
 /**
  * Custom harness for interacting with the filter modal component in tests.
  */
 export class FilterModalHarness extends SkyComponentHarness {
   public static hostSelector = '.filter-modal-demo-form';
 
-  #getSelectElement = this.locatorFor(
-    'select[formControlName="selectedOption"]',
-  );
   #getSaveButton = this.locatorFor('button.sky-btn-primary');
   #getCancelButton = this.locatorFor('button.sky-btn-link');
+  #getRadioGroup = this.locatorFor(SkyRadioGroupHarness);
 
   public static with(): HarnessPredicate<FilterModalHarness> {
     return new HarnessPredicate(FilterModalHarness, {});
   }
 
   /**
-   * Selects an option and saves the modal.
+   * Select a radio option by its 0-based index within the group and save.
```

Comment:
```suggestion
   * Selects a radio button based on its 0-based index within the group.
```
I'm not completely sure what "and save" means at the end here. It seems like stray words, but if we are saying that this both selects the radio button and also saves that state, I think we probably need a few more words here. I'm happy to help with that if necessary.